scss/book-list-detail.scss

```diff
@@ -0,0 +1,60 @@
+.body{
+  font-family: 'Roboto', sans-serif;
+}
+
+.master-container{
+  display: flex;
+  height: 768px;
+  width: 1366px;
```

Comment:
This is way too fixed. The heigth should be `auto` so it will grow with the content. The width, it's ok to set a max width, but let's add flexibility to it by changing the width to:
```css
max-width: 1366px;
width: 100%;
```
That way it will grow or shrink based on the size of the display 😄